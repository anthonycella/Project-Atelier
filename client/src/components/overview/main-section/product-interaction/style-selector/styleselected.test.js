import renderer from 'react-test-renderer';
import StyleSelected from './styleselected.jsx';

it('Style Selected renders to the page', () => {
  const name = 'larry';
  const colorScheme = { textColorBackground: 'black' };

  const component = renderer.create(
    <StyleSelected name={name} colorScheme={colorScheme} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
