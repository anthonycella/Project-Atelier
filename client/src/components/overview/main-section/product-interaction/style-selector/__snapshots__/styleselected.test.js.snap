// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Style Selected renders to the page 1`] = `
<div
  className="sc-dkzDqf gUCSvl"
>
  <span
    className="sc-bczRLJ evRwiw"
  >
    Style 
    &gt;
     
  </span>
  <span
    className="sc-gsnTZi iaCvAC"
  >
    larry
  </span>
</div>
`;
